src/atoms/affine/multiply_divide.jl

```diff
@@ -205,5 +205,5 @@ function broadcast(::typeof(*), x::AbstractExpr, y::AbstractExpr)
 end
 broadcast(::typeof(*), x::Value, y::AbstractExpr) = DotMultiplyAtom(Constant(x), y)
 broadcast(::typeof(*), x::AbstractExpr, y::Value) = DotMultiplyAtom(Constant(y), x)
-broadcast(::typeof(/), x::AbstractExpr, y::Value) = DotMultiplyAtom(Constant(1./y), x)
+broadcast(::typeof(/), x::AbstractExpr, y::Value) = DotMultiplyAtom(Constant(1 ./ y), x)
```

Comment:
I believe instead of overloading `Base.broadcast` directly, we're now supposed to go through a different set of machinery laid out in https://docs.julialang.org/en/v1/manual/interfaces/#man-interfaces-broadcasting-1. For now this is probably okay, since it does technically work to do it this way, but I can look into reworking this later.